_pages/saml/authentication.md

```diff
@@ -114,8 +174,10 @@ A proofed identity request at AAL2, with phishing resistent MFA, for email, phon
 <div class="grid-row grid-gap">
     <div class="desktop:grid-col-7 mobile:grid-col-full">
         <h2 id="authentication-response">Authentication response</h2>
-        <p>After the user authenticates, Login.gov will redirect and POST a form back to your registered Assertion Consumer Service URL:</p>
-        <p>The SAMLResponse is a base64-encoded XML payload that contains encrypted data.</p>
+        <p>After the user authenticates, Login.gov will redirect and POST a form back to your registered Assertion Consumer Service URL with a hidden form control named SAMLResponse.</p>
+        <p>The `SAMLResponse` is a base64-encoded XML payload that contains data that is encrypted with the service provider's public key.</p>
+        <p>{{ response_elements | markdownify }}</p>
+        <p>For example, {{ decrypted_response | markdownify }}</p>
```

Comment:
markdown inside HTML is not automatic but you can do:
```suggestion
        <div markdown="1">
## markdown content here
        </div>
```

Reply:
Thank you, that was what I was looking for!